packages/jest-snapshot/src/index.ts

```diff
@@ -26,6 +31,37 @@ type Context = MatcherState & {
   snapshotState: SnapshotState;
 };
 
+type MatchSnapshotConfig = {
+  context: Context;
+  expectedArgument: string;
+  inlineSnapshot?: string;
+  matcherName: string;
+  options: MatcherHintOptions;
+  propertyMatchers?: any;
+  received: any;
+  testName?: string;
+};
+
+const DID_NOT_THROW = 'Received function did not throw'; // same as toThrow
+const NOT_SNAPSHOT_MATCHERS = '.not cannot be used with snapshot matchers';
```

Comment:
I wonder if we should use `".not"` instead of `.not` so it's more distinguishable?

Reply:
or at least wrap it in `chalk.bold`?

Reply:
<img width="600" alt="toMatchInlineSnapshot not improved 2" src="https://user-images.githubusercontent.com/11862657/54492897-dd85a000-48a0-11e9-9247-6e3c4c00231e.png">